src/lib.rs

```diff
@@ -1,10 +1,15 @@
 //! NMEA 0183 parser
 //!
-//! Use nmea::Nmea::parse and nmea::Nmea::parse_for_fix to preserve
-//! state between recieving new nmea sentence, and nmea::parse
-//! to parse sentences without state
+//! [![Version](https://img.shields.io/crates/v/nmea.svg)](https://crates.io/crates/nmea)
+//! [![Build Status](https://github.com/AeroRust/nmea/workflows/CI/badge.svg)](https://github.com/AeroRust/nmea/actions?query=workflow%3ACI+branch%3Amaster)
+//! [![codecov](https://codecov.io/gh/AeroRust/nmea/branch/master/graph/badge.svg)](https://codecov.io/gh/AeroRust/nmea)
+//! [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/AeroRust/nmea/blob/master/LICENSE.txt)
```

Comment:
Why we need all this links above? Is it really what user need in docs.rs?

Reply:
I would imagine License codecov to be a good addition even to the docs but I can remove all if it's a problem.

I can remove the version & build status.

Reply:
It is  notthe problem, but docs.rs, based on my experience, is the last step for user.
At first crate would be found on github/crates.io/lib.rs where he/she already read README,
and only then he/she comes to docs.rs and here the same information that potential user already read, while user expect documentation about functions/struct and so on, not license information or status of coverage.

Reply:
Ok, let me remove them then.